sdks/java/extensions/join-library/README.md

```diff
@@ -22,12 +41,4 @@ Example how to use join-library:
     PCollection<KV<String, KV<String, Long>>> joinedPcollection =
       Join.innerJoin(leftPcollection, rightPcollection);
 
-Join-library can be found on maven-central:
-
-    <dependency>
-      <groupId>org.linuxalert.dataflow</groupId>
-      <artifactId>google-cloud-dataflow-java-contrib-joinlibrary</artifactId>
-      <version>0.0.3</version>
-    </dependency>
-
```

Comment:
@jbonofre is this going to be deployed as part of our release? If not, should we ask @mrunesson to deploy it himself again (under a `beam`-related pacakge name)?